TL improvement (#382)
07th-mod/higurashi-console-arcs: Update/omot_026.txt

			NULL, "\"I was injured in the line of duty!", Line_WaitForInput);
	ModPlayVoiceLS(4, 11, "ps3/s14/11/220700475", 256, TRUE);
	OutputLine(NULL, "　ちょっとは思いやりのある治療をしていただいてもバチは当たらないと思うんですけどねぇ？」",

> 			NULL, " So even if I ask you for a little more caring treatment, there's no problem, right?\" ", Line_Normal);

Reviewer: ```suggestion
			NULL, " So there's nothing wrong with asking for a little more caring treatment, right?\" ", Line_Normal);
```

Reviewer 2: ```suggestion
			NULL, " So there's nothing wrong with asking for a little more caring treatment, right?\" ", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#797d8a>赤坂</color>", NULL, "<color=#797d8a>Akasaka</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 13, "ps3/s14/13/221300138", 256, TRUE);
	OutputLine(NULL, "「月並みだが、迷ってる時間が惜しい。",

> 			NULL, "\"I know I'm saying something banal but, there's no time to make slow decicion.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"I know this is cliche, but there's no time to waste making slow decicions.", Line_WaitForInput);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#5ec69a>詩音</color>", NULL, "<color=#5ec69a>Shion</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 6, "ps3/s14/06/220600257", 256, TRUE);
	OutputLine(NULL, "「…言うのはただですけどね～？」",

> 			NULL, "\"...it's easy just to tell, huh~?\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"...It's easy to just say that though, huh~?\"", Line_Normal);
```

			NULL, " In the end, I couldn't even step foot on the battlefield.", Line_WaitForInput);
	ModPlayVoiceLS(4, 6, "ps3/s14/06/220600261", 256, TRUE);
	OutputLine(NULL, "なんだか、性に合わないことしているなぁ」",
			NULL, " I wonder if that's more appropriate for a girl like me.\"", Line_Normal);
	ClearMessage();

//r赤坂さんの頭に絆創膏を貼り付けながら、なんだか今の自分がおかしくて愚痴が出る。本当はお姉や圭ちゃんたちみたく、鉄火場に乗り込んで大暴れするほうが私らしくて気分がいいのだけど。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "赤坂さんの頭に絆創膏を貼り付けながら、なんだか今の自分がおかしくて愚痴が出る。本当はお姉や圭ちゃんたちみたく、鉄火場に乗り込んで大暴れするほうが私らしくて気分がいいのだけど。",

> 			NULL, "Somehow, I found myself making weird complaints as I pasted a bandage to Akasaka-san's head. Actually, I'd feel better going on a rampage on the battlefield like Sis and Kei-chan and it suits me though.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Somehow, I found myself making weird complaints as I pasted a bandage to Akasaka-san's head. Really, I'd feel better going on a rampage on the battlefield like Sis and Kei-chan, and it would suit me.", Line_Normal);
```

			NULL, " ...I was just thinking that treating the injured really suits you.", Line_WaitForInput);
	ModPlayVoiceLS(4, 13, "ps3/s14/13/221300142", 256, TRUE);
	OutputLine(NULL, "どうだい、ナースさんを本気で目指してみたら？」",

> 			NULL, " Hey, why don't you become a nurse?\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " Hey, why not try becoming a nurse?\"", Line_Normal);
```
